index.html

```diff
@@ -50,7 +50,197 @@ <h1 class="primary-text">
           </p>
         </div>
         <div class="blank"></div>
-      </div>
+      </section>
+
+      <section class="works">
+        <div class="wrapper">
+          <div class="content">
+            <div class="title">
+              <h1>My Recent Works</h1>
+              <span></span>
+            </div>
+            <div class="card-works">
+              <div class="card-left">
+                <img src="img/ImgPlaceholder.png" alt="Image placeholder">
+              </div>
+              <div class="card-right">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <div>
+                  <p>css</p>
+                  <p>html</p>
+                  <p>bootstrap</p>
+                  <p>ruby</p>
+                </div>
```

Comment:
- [ ] Kindly group the list of `programming languages` in a `ul` tag and each of them in an `li` tag. This would make your code more semantic 🙏